src/comm.jl

```diff
@@ -95,3 +95,10 @@ function Intercomm_merge(intercomm::Comm, flag::Bool)
     return newcomm
 end
 
+function Universe_size(comm::Comm)
```

Comment:
Since this doesn't correspond exactly to an MPI function, maybe make it lowercase?

```suggestion
function universe_size(comm::Comm)
```